torch/_dynamo/variables/builder.py

```diff
@@ -2647,6 +2647,7 @@ def handle_traced_output(example_value, tx, proxy, options, subclass_type, targe
         proxy.node.target
         in [
             torch._C._are_functorch_transforms_active,
+            torch._C._functorch.is_batchedtensor,
```

Comment:
do you have some more context over what the problem was?

Reply:
`is_batchedtensor` returns a boolean value and it is not in the list of allowed functions on builder.py